database/seeders/DatabaseSeeder.php

```
@@ -24,6 +24,12 @@ public function run(): void
         if (\App\Models\ExpenseCategory::count() === 0) {
             $this->call(ExpenseCategorySeeder::class);
         }
+        // Permissions are global, not tenant-scoped — this count is never
+        // affected by how many tenants exist. Re-run any time the catalog
+        // gains a permission; firstOrCreate() never touches existing rows.
+        if (\Spatie\Permission\Models\Permission::count() < count(\App\Authorization\PermissionCatalog::names())) {
```

PHPStan (2.2.8)

[error] Call to an undefined static method Spatie\Permission\Models\Permission::count().


(staticMethod.notFound)